src/main/java/uk/gov/cabinetoffice/csl/domain/User.java

```diff
@@ -27,7 +27,7 @@ public class User implements Serializable {
     private String gradeName;
     private String lineManagerName;
     private String lineManagerEmail;
-    private ArrayList<BasicOrganisationalUnit> departmentHierarchy = new ArrayList<>();
```

Comment:
Can we keep this please? It seems to add a lot more code for null checking etc. removing it and I don't think it's bad parctice to keep it there personally